src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar">
```
![image](https://user-images.githubusercontent.com/38065883/131717679-e3823cdc-2ede-48bc-898a-0a37766c5181.png)
Добавь своему элементу такие классы и проверь все ли работает правильно